.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - "0.10"
-  - "0.11"
+  - "0.12"
```

Comment:
Dropping support for old Node versions and bumping up the major version is out of scope of this patch.

I am also not sure if can do that yet, because loopback-workspace requires this module and AFAIK, we need loopback-workspace to keep supporting pre-4 versions of Node because APIC supports 0.12 (or at least used to do).

Anyways, this modules changes so infrequently, that it supporting few more legacy Node.js versions is not a problem.